<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>401</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt</string>
   </property>
  </widget>
  <widget class="QListWidget" name="tasksListWidget">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>150</y>
     <width>301</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>450</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16pt;
background:  #01BFFF;
color:  white;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Save changes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addNewButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>110</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:  #01BFFF;
color:  white;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Add new</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 24pt;
background:  #01BFFF;
color:  white;
border-radius: 8px;</string>
   </property>
   <property name="text">
    <string>Daily Task Planner</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="taskLineEdit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>110</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
